src/tools/perf/perftest_mad.c

```diff
@@ -0,0 +1,695 @@
+/**
+* Copyright (c) NVIDIA CORPORATION & AFFILIATES, 2023. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include "perftest_mad.h"
+#include "perftest.h"
+
+#include <ucs/sys/string.h>
+#include <ucs/sys/sys.h>
+#include <ucs/sys/sock.h>
+#include <ucs/debug/log.h>
+#include <ucs/sys/iovec.inl>
+
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <netdb.h>
+#include <sys/poll.h>
+#include <linux/types.h> /* __be64 */
+
+#include <infiniband/mad.h>
+#include <infiniband/umad.h>
+#include <infiniband/umad_types.h>
+
+#define PERFTEST_RTE_CLASS      (IB_VENDOR_RANGE2_START_CLASS + 0x10)
+#define PERFTEST_RTE_OPENIB_OUI IB_OPENIB_OUI
```

Comment:
why do we need to re-define this constant?

Reply:
removed the intermediate definition